src/test/java/racingcar/model/RuleTest.java

```diff
@@ -0,0 +1,33 @@
+package racingcar.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RuleTest {
+    @Test
+    @DisplayName("글자수가 5자 이하일 때 글자수 제한 오류 발생 여부")
+    void NonNameLenghtError() {
+        //Given
+        Rule rule = new Rule();
+        String carName = "carA";
+
+        //When & Then
+        assertDoesNotThrow(() -> rule.CarNameLengthLimit(carName));
+    }
+
+    @Test
+    @DisplayName("글자수가 n자 초과일 때 글자수 제한 오류 발생 여부")
+    void nameLenghtError() {
+        //Given
+        Rule rule = new Rule();
+        String carName = "carABC";
+
+        //When & Then
+        IllegalArgumentException exception = assertThrows(IllegalArgumentException.class, () -> rule.CarNameLengthLimit(carName));
+
+        //Then
+        assertEquals("자동차 이름은 5자 이하만 가능합니다. 프로그램을 종료합니다.", exception.getMessage());
```

Comment:
P3)
이 부분은 아래처럼 더 깔끔하게 만들 수도 있어요

```suggestion
        //When & Then
        Assertions.assertThatThrownBy(() -> rule.CarNameLengthLimit(carName))
                .isInstanceOf(IllegalArgumentException.class)
                .hasMessage("자동차 이름은 5자 이하만 가능합니다. 프로그램을 종료합니다.");
```

Reply:
수정하였습니다!
훨씬 코드도 깔끔하고 의미도 파악하기 편한 방법인 것 같아요:)